packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -123,7 +124,9 @@ async function handleDeleteContext(contextName: string) {
                 {:else}
                   <div class="flex flex-row pt-2">
                     <div class="w-3 h-3 rounded-full bg-gray-900"></div>
-                    <div class="ml-1 font-bold text-[9px] text-gray-900">UNREACHABLE</div>
+                    <div class="ml-1 font-bold text-[9px] text-gray-900" aria-label="context-unreachable">
```

Comment:
We have agreed that aria label should have this `Context Unreachable` notation. Please update in the whole file.

Reply:
resolved [here](https://github.com/containers/podman-desktop/pull/6791)